### ISSUE TITLE
predict for imagenet categories

### ISSUE THREAD

**Issue by kzhang3256** [pseeth/soundnet_keras#4]
How can we use this code to predict object categories?
The original code 
http://soundnet.csail.mit.edu/ 
can predict object and scene at the same time.

Thanks!

**Comment by benlawson** [comment]
yeah the weights are saved in the same file. I had to switch to the keras functional api to get it working (see my PR here #5)
